src/pages/billing/SpendReport/SpendReport.ts

```diff
@@ -324,12 +336,14 @@ export const SpendReport = (props: SpendReportProps) => {
             (name) =>
               h(CostCard, {
                 type: name,
-                title: `Total ${name}`,
+                title: `Total ${props.cloudPlatform === 'GCP' ? name : azureCategoryCardCaptionMap.get(name)}`,
```

Comment:
I would suggest you just pass the `name` to a helper function that possibly transforms it (or else just returns the input `name` for anything other than 'workspaceinfrastructure`). That would allow you to avoid this platform check, as the method would not need to be platform-specific.

Reply:
Agree that helper function would be better solution for this. But I probably can't just return name for anything except 'workspaceinfrastructure'. Mapping for Azure spend report is not straightforward. That's why I added specific variable which holds this mapping.